exersice_html/home.html

```diff
@@ -0,0 +1,285 @@
+        </div>
+        <div class="search-group">
+          <span class="search-icon">
+            <i class="fa fa-search"></i>
+          </span>
+          <input class="search" placeholder="Search" />
+        </div>
+      </div>
+      <div class="header-right">
+        <a href="#" class="cart">
+          <i class="fa fa-shopping-cart"></i>
+          <span class="cart-amount">4</span>
+        </a>
+        <a href="#" class="sign-in">Sign in</a>
+        <a href="#" class="btn btn-primary">Sign up</a>
+      </div>
+    </div>
+    </div>
+  </header>
+  <main>
+      <section>
+        <div class="container banner">
+          <img class="banner-img" src="./image/banner.png" alt="banner" />
+        </div>
+      </section>
+      <section>
+        <div class="container">
+          <div class="toolbar">
+            <div class="sort">
+              <select class="sort-uselessFirst">
+                <option>Useless first</option>
+              </select>
+              <select class="sort-condition">
+                <option>Condition</option>
+              </select>
+              <select class="sort-delivery">
+                <option>Delivery option</option>
+              </select>
+            </div>
+            <div class="action">
+              <div class="action-group">
+                <div class="btn action-btn">
+                  <label for="show-all">Show all</label>
+                  <input type="radio" name="action" id="show-all"/>
+                </div>
+                <div class="btn action-btn">
+                  <label for="auction" class="btn action-btn">Auction</label>
+                  <input type="radio" name="action" id="auction"/>  
+                </div>
+                <div class="btn action-btn">
+                  <label for="buy-now" class="btn action-btn">Buy now</label>
+                  <input type="radio" name="action" id="buy-now"/> 
+                </div>  
+              </div>
+              <div class="type-show">
+                <div class="btn action-btn">
+                  <label for="show-all"><i class="fa fa-list"></i></label>
+                  <input type="radio" name="action" id="show-all"/>
+                </div>
+                <div class="btn action-btn">
+                  <label for="auction" class="btn action-btn"><i class="fa fa-th-large"></i></label>
+                  <input type="radio" name="action" id="auction"/>  
+                </div>
+              </div>
+            </div>
+          </div>
+          <ul class="grid-list products">
+            <li class="grid-item product">
+              <a href="#" class="product-info">
+                <img class="product-img" src="./image/Product-1.png" alt="product" />
+                <h4 class="product-content">
+                  Vintage Typewriter to post awesome stories about UI design and
+                  webdev.
+                </h4>
+                <div class="product-info-group">
```

Comment:
Consider other class name to make sense.
- product-name
- product-price
- product-description
...